protos/orderer/smartbft/configuration.proto

```diff
@@ -25,25 +25,14 @@ message Consenter {
     uint32 consenter_id = 1;
     string host = 2;
     uint32 port = 3;
-    bytes identity = 4; // consenter serialized identity
-    bytes client_tls_cert = 5;
-    bytes server_tls_cert = 6;
+    string msp_id = 4;
+    bytes identity = 5; // consenter serialized identity
```

Comment:
this isn't the serialized identity anymore, but the certificate, no?

Reply:
This is serialized identity.